betterlockscreen

```diff
@@ -95,7 +61,7 @@ init_config () {
     DEFAULT_DPMS=$(xset q | awk '/^[[:blank:]]*DPMS is/ {print $(NF)}')
 
     # Dunst
-    DUNST_INSTALLED=false && [[ -e "$(command -v dunstctl)" ]] && DUNST_INSTALLED=true
+    DUNST_INSTALLED=false && [[ -e "$(command -v dunstctl)" && "$dunst_enabled" == "true" ]] && DUNST_INSTALLED=true
```

Comment:
This is not working for me. I still see dbus errors.

Reply:
#284 was just meant to introduce the variable which makes it possible for advanced users to enable/disable dunst based on an expression. The default it would be true for users who don't need this features.

Reply:
So it does not break the current behavior